datameta/static/js/view.js

```diff
@@ -23,25 +23,32 @@ DataMeta.view.buildColumns = function(mdata) {
         return {
             title : mdatum.serviceId === null ? mdatum.name : '<i class="bi bi-cpu"></i> '+mdatum.name,
             data : null,
-            render : function(mdataset, type, row, meta) {
+            render : function(mdataset, type, row, meta) {                              
                 // We don't have access
                 if (!(mdatum.name in mdataset.record)) return '<i class="bi bi-lock-fill text-danger"></i>';
                 // Special case NULL and service metadatum with access but not run yet
                 if (mdataset.serviceExecutions !== null
                     && mdatum.name in mdataset.serviceExecutions
-                    && mdataset.serviceExecutions[mdatum.name]===null)
+                    && mdataset.serviceExecutions[mdatum.name] === null)
                     return '<span class="text-black-50"><i class="bi bi-hourglass-split"></i> <i>pending</i></span>';
                 // Special case NULL
                 if (mdataset.record[mdatum.name] === null) return '<span class="text-black-50"><i>empty</i></span>';
-                // Speical case file
-                if (mdataset.fileIds[mdatum.name]) return '<a class="link-bare" href="' + DataMeta.api('rpc/get-file-url/'+mdataset.fileIds[mdatum.name].site) +'?redirect=true"><i class="bi bi-cloud-arrow-down-fill"></i> '+mdataset.record[mdatum.name]+'</a>';
+                // Special case file
+                if (mdataset.fileIds[mdatum.name]) {
+                    var record_str = mdataset.replacedBy === null ? mdataset.record[mdatum.name] : '<span class="large-super" style="color:#ccc"><i>' + mdataset.record[mdatum.name] + '</i></span>';
+                    return '<a class="link-bare" href="' + DataMeta.api('rpc/get-file-url/'+mdataset.fileIds[mdatum.name].site) + '?redirect=true"><i class="bi bi-cloud-arrow-down-fill"></i>' + record_str + '</a>';
```

Comment:
Space between the cloud and the filename got lost here :)
```suggestion
                    return '<a class="link-bare" href="' + DataMeta.api('rpc/get-file-url/'+mdataset.fileIds[mdatum.name].site) + '?redirect=true"><i class="bi bi-cloud-arrow-down-fill"></i> ' + record_str + '</a>';
```